client/src/components/dashboard/Games.js

```diff
@@ -44,8 +43,8 @@ export default class Games extends React.Component {
     this.handlePageClick = this.handlePageClick.bind(this)
     this.handleRefresh = this.handleRefresh.bind(this)
   }
-  //Need to slice up the array to display certain amount per page
-  //I'm not really too sure on the proper way to call and modify parameters inside of a function
+  // Need to slice up the array to display certain amount per page
```

Comment:
This comment should probably get bopped since it was when we were trying to slice up the array, it's not really applicable now.